src/commands/bundle.js

```diff
@@ -66,7 +66,15 @@ async function bundle(opts: *) {
   const stats = await new Promise((resolve, reject) =>
     compiler.run((err, info) => {
       if (err || info.hasErrors()) {
-        reject(new MessageError(messages.bundleFailed()));
+        reject(
+          new MessageError(
+            messages.bundleFailed({
+              errors: err
+                ? [err.toString()]
```

Comment:
If `err` is `Error` instance, how about using `err.message`? We drop `Error: ` prefix and just receive error message

Reply:
Makes sense!